index.html

```diff
@@ -7,5 +7,42 @@
     <link rel="stylesheet" href="style.css" />
     <title>Portfolio</title>
   </head>
-  <body></body>
+  <body>
+    <header id="header">
+      <div id="owner_name">
+        <p>Derbal Amine</p>
+      </div>
+      <div id="header_menu">
+        <img src="images/icons8-menu-32.png" alt="header_menu" width="20px" />
+      </div>
+    </header>
+    <section id="headline">
+      <div id="introduction">
+        <p>Hey There.</p>
+        <p>I'm Amine</p>
+      </div>
+      <div id="Occupation">I'am a Software Developer</div>
+      <div id="qualification">
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </div>
+      <nav id="contact">
+        <ul id="contact_infos">
+          <li><img src="images/github_icon.png" alt="github" /></li>
+          <li>
+            <img
+              src="images/linkedin_icon_part_1.png"
+              alt="linkedin"
+              srcset=""
+            /><img src="images/linkedin_icon.png" alt="linkedin" />
+          </li>
+          <li><img src="images/contact_icon.png" alt="contact" /></li>
+          <li><img src="images/twiter_icon.png" alt="twiter" /></li>
+          <li><img src="images/mail_icon.png" alt="email" /></li>
+        </ul>
+      </nav>
+      <div id="illustration"><img src="images/Illustration.png" alt="" /></div>
```

Comment:
- [ ] Kindly make sure that the section has the proper background image (decorations are always background images, not `img` tags).If an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.